nib.gemspec

```diff
@@ -23,6 +23,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency('gli', '~> 2.16')
 
   s.add_development_dependency('codeclimate-test-reporter', '~> 1.0.7')
+  s.add_development_dependency('did_you_mean')
```

Comment:
Hopefully this can be removed in the long run: https://github.com/rubocop-hq/rubocop/issues/7979